src/Symfony/Component/Validator/Tests/Constraints/PasswordStrengthValidatorWithClosureTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Constraints;
+
+use Symfony\Component\Validator\Constraints\PasswordStrength;
+use Symfony\Component\Validator\Constraints\PasswordStrengthValidator;
+use Symfony\Component\Validator\Password\PasswordStrengthEstimator;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
+use Symfony\Component\Validator\Tests\Constraints\Fixtures\StringableValue;
+
+class PasswordStrengthValidatorWithClosureTest extends ConstraintValidatorTestCase
```

Comment:
These tests triggers deprecated code paths. We need to add them to the `@group legacy`, otherwise our CI won't accept the triggered deprecation warnings.

Reply:
I applied the @group legacy annotation to the test class (see last commit) and it was ok in my local runs 
```bash
./phpunit src/Symfony/Component/Validator
``` 
but it still fails here in the github actions => how should this be applied?